packages/flutter/lib/src/cupertino/nav_bar.dart

```diff
@@ -109,6 +115,15 @@ class CupertinoNavigationBar extends StatelessWidget implements ObstructingPrefe
   /// behind it.
   final Color backgroundColor;
 
+  /// The bottom border color of the navigation bar.
+  ///
+  /// If a border color is null or transparent, the tab bar will not display a border.
+  final Color borderColor;
```

Comment:
If we add a new configurable, I'd err on the side of flexibility and let the user specify a `Border bottomBorder` instead of a `Color borderColor` today and maybe a `double borderWidth` tomorrow etc.

Reply:
Agree, was debating, because of background color I think. Will change.